internal/langserver/handlers/execute_command_module_providers_test.go

```diff
@@ -0,0 +1,168 @@
+package handlers
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/creachadair/jrpc2/code"
+	"github.com/hashicorp/go-version"
+	"github.com/hashicorp/terraform-ls/internal/langserver"
+	"github.com/hashicorp/terraform-ls/internal/langserver/cmd"
+	"github.com/hashicorp/terraform-ls/internal/state"
+	"github.com/hashicorp/terraform-ls/internal/terraform/exec"
+	"github.com/hashicorp/terraform-ls/internal/uri"
+	tfaddr "github.com/hashicorp/terraform-registry-address"
+	tfmod "github.com/hashicorp/terraform-schema/module"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestLangServer_workspaceExecuteCommand_moduleProviders_argumentError(t *testing.T) {
+	rootDir := t.TempDir()
+	rootUri := uri.FromPath(rootDir)
+
+	ls := langserver.NewLangServerMock(t, NewMockSession(&MockSessionInput{
+		TerraformCalls: &exec.TerraformMockCalls{
+			PerWorkDir: map[string][]*mock.Call{
+				rootDir: validTfMockCalls(),
+			},
+		},
+	}))
+	stop := ls.Start(t)
+	defer stop()
+
+	ls.Call(t, &langserver.CallRequest{
+		Method: "initialize",
+		ReqParams: fmt.Sprintf(`{
+	    "capabilities": {},
+	    "rootUri": %q,
+		"processId": 12345
```

Comment:
```suggestion
		"processId": 12345
```
Nitpick - this looks a bit misaligned, I think there's a mixture of spaces and tabs here.

Reply:
I guess gofmt won't format things here, because it's inside a string?

Reply:
Yes, that is one of the reasons, but also this is JSON, not Go, so I'm not sure gofmt would be able to handle it anyway. It doesn't need to be formatted the same throughout the codebase, although if you wanted to take care of that (in a separate PR) that would be nice! 😄 I was more referring to the fact that it stands out from the diff here in this particular test.